packages/flutter/lib/src/widgets/_html_element_view_io.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: prefer_const_constructors_in_immutables
+// ignore_for_file: avoid_unused_constructor_parameters
+
+import 'framework.dart';
+import 'platform_view.dart';
+
+/// The platform-specific implementation of [HtmlElementView].
+extension HtmlElementViewImpl on HtmlElementView {
```

Comment:
Can this extension be made private `_HtmlElementViewImpl` (and also the methods it defines)? That way we don't pollute the Flutter API docs with the implementation details for the web platform view.

Reply:
If it's made private, it can't be used from `src/widgets/platform_view.dart`. IIUC, files that start with `_` are considered private and won't be exported outside of flutter.

Reply:
Ahhh yes you're right!